lib/index.js

```diff
@@ -316,6 +316,54 @@ exports.parseSchemaOrgMicrodata = BBPromise.method(function(chtml){
 });
 
 /**
+ * Scrapes Highwire Press metadata given html object
+ * @param  {Object}   chtml html Cheerio object
+ * @return {Object}   promise of highwire press metadata object
+ */
+exports.parseHighwirePress = BBPromise.method(function(chtml){
+
+	var meta = {};
+	var metaTags = chtml('meta');
+
+	if (!metaTags || metaTags.length === 0) {
+		return BBPromise.reject('No Highwire Press metadata found in page');
+	}
+
+	metaTags.each(function() {
+		var element = chtml(this),
+			nameAttr = element.attr('name');
+
+		// If the element isn't a Highwire Press property, skip it
+		if (!nameAttr
+			|| (nameAttr.substring(0, 9).toUpperCase() !== 'CITATION_')) {
+			return;
+		}
+
+		var property = nameAttr.substring(nameAttr.indexOf('_') + 1),
+			content = element.attr('content');
```

Comment:
maybe content should be to lower case in case the cases are mixed? unlikely but might as well as these aren't case sensitive.